new_tests/test_extract.py

```diff
@@ -107,48 +78,58 @@ def test_empty(self):
+                                         'outputs': {'.tsv': 3},
+                                         'ravel_piano': {'.mscx': 5, '.tsv': 14},
+                                         'sweelinck_keyboard': {'.mscx': 1, '.tsv': 4},
+                                         'wagner_overtures': {'.mscx': 2, '.tsv': 7}},
+            "parsed_all-regular_dirs": {'ravel_piano': {'.mscx': 5, '.tsv': 14},
+                                         'sweelinck_keyboard': {'.mscx': 1, '.tsv': 4},
+                                         'wagner_overtures': {'.mscx': 2, '.tsv': 7}},
+            "parsed_mscx-chaotic_dirs": {'mixed_files': {'.mscx': 8, '.mscz': 1, '.tsv': 1}, 'outputs': {'.tsv': 3}},
+            "parsed_mscx-everything": {'mixed_files': {'.mscx': 8, '.mscz': 1, '.tsv': 1},
+                                         'outputs': {'.tsv': 3},
+                                         'ravel_piano': {'.mscx': 5, '.tsv': 14},
+                                         'sweelinck_keyboard': {'.mscx': 1, '.tsv': 4},
+                                         'wagner_overtures': {'.mscx': 2, '.tsv': 7}},
+            "parsed_mscx-regular_dirs": {'ravel_piano': {'.mscx': 5, '.tsv': 14},
+                                         'sweelinck_keyboard': {'.mscx': 1, '.tsv': 4},
+                                         'wagner_overtures': {'.mscx': 2, '.tsv': 7}},
+            "parsed_tsv-chaotic_dirs": {'mixed_files': {'.mscx': 8, '.mscz': 1, '.tsv': 1}, 'outputs': {'.tsv': 3}},
+            "parsed_tsv-everything": {'mixed_files': {'.mscx': 8, '.mscz': 1, '.tsv': 1},
+                                         'outputs': {'.tsv': 3},
+                                         'ravel_piano': {'.mscx': 5, '.tsv': 14},
+                                         'sweelinck_keyboard': {'.mscx': 1, '.tsv': 4},
+                                         'wagner_overtures': {'.mscx': 2, '.tsv': 7}},
+            "parsed_tsv-regular_dirs": {'ravel_piano': {'.mscx': 5, '.tsv': 14},
+                                         'sweelinck_keyboard': {'.mscx': 1, '.tsv': 4},
+                                         'wagner_overtures': {'.mscx': 2, '.tsv': 7}},
+        })
         name = node_name2cfg_name(request.node.name)
         return name2expected[name]
 
+    @pytest.fixture()
+    def get_ignored_warnings(self, filter_path='unittest_metacorpus/mixed_files'):
+        # get warnings from file
+        with open(os.path.join(str(Path.home()), filter_path, 'IGNORED_WARNINGS')) as f:
+            ignored_messages = f.read()
+        return ignored_messages
+
     def test_keys(self, expected_keys):
         assert self.parsed_parse_obj.count_extensions(per_key=True) == expected_keys
 
+    def test_check(self, caplog, get_ignored_warnings):
+        _ = self.parsed_parse_obj.get_dataframes(expanded=True)
+        for record in caplog.records:
+            if " ".join(record.getMessage().split(sep="--")[0].split(sep=" ")) in get_ignored_warnings:
```

Comment:
Testing the string is certainly an option we can keep up, but through we also have access to the logger information and can test the level directly.